tests/speed_test/speed_test.py

```diff
@@ -4,17 +4,36 @@
 from time import time
 
 from tabulate import tabulate
+from tokenizers import BPETokenizer
```

Comment:
I would replace to `from tokenizers import BPETokenizer as HuggingFaceBPETokenizer`